components/src/core/drawer/drawer-header/drawer-header.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { DrawerHeaderComponent } from './drawer-header.component';
+import { MatDividerModule } from '@angular/material/divider';
+import { MatToolbarModule, MatIconModule, MatButtonModule } from '@angular/material';
+
+@NgModule({
+    declarations: [DrawerHeaderComponent],
+    imports: [
+        CommonModule,
+        MatDividerModule,
+        MatToolbarModule,
+        MatIconModule,
+        MatButtonModule
```

Comment:
MatButtonModule and MatDividerModule can be removed.

Reply:
Removed